src/frontend/api.ts

```diff
@@ -154,15 +167,55 @@ const SERVER_START_TIMEOUT = 30000;
 
 const pixelRatio = PixelRatio.get();
 
-export function Api({
-  baseUrl,
-  timeout = DEFAULT_TIMEOUT,
-}: {
-  baseUrl: string,
-  timeout?: number,
-}) {
+interface PublicApi {
```

Comment:
Nit-pick: I feel like this is easier to maintain when we define these types inline above the functions that they refer to. Typescript will imply the same of the exported api.

Reply:
Yeah i implemented this interface initially to get a better idea of the API type, but looking back it's definitely harder to work with than just inlining the annotations and taking advantage of TS inference. Just updated to fix 👍